.github/workflows/backup-20250905-114531/ci-with-codecov.yml

```
@@ -0,0 +1,72 @@
+      - name: Build
+        run: pnpm nx affected --target=build --parallel=3
+      
+      - name: Test with Coverage
+        run: pnpm nx run-many --target=test --all --coverage
+      
+      - name: Create combined coverage report
+        run: |
+          mkdir -p ./coverage/combined
+          npx nyc merge ./coverage ./coverage/combined/coverage.json
+          npx nyc report --reporter=lcov --report-dir=./coverage/combined --temp-dir=./coverage/combined
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)